model/countries/AU/AU-parsing-rules.yaml

```diff
@@ -0,0 +1,232 @@
+regex_definitions:
+  kBuildingValueRe:  # Regex for type "building"
+    regex_concat:
+      parts:
+      - regex_fragment: \d+  # House number
+      # Optional suffix for building ids with a letter suffix, e.g. 12a.
+      - regex_fragment: \w?
+      wrap_non_capture: false
```

Comment:
I think the documentation also references building values base on ranges  (i.e. 17-19). You might want to look at the DE parsing expression on how to handle those.